src/option_parser.c

```diff
@@ -202,6 +202,7 @@ char *clean_value(char *value, int line_num)
                         switch (*unparsed) {
                         case '\\': /* verbatim */ break;
                         case '"': /* verbatim */ break;
+                        case 'n': *unparsed = '\n'; break;
```

Comment:
Same for this, only one command per line and there shouldn't be anything else on the same line as a switch case label.